docs/fields/array.mdx

```diff
@@ -80,7 +80,7 @@ export const MyArrayField: Field = {
 }
 ```
 
-The Array Field inherits all of the default options from the base [Field Admin Config](./overview#admin-options), plus the following additional options:
+The Array Field inherits all of the default admin options from the base [Field Admin Config](./overview#admin-options), plus the following additional options:
```

Comment:
Adding "admin" here is redundant is it not?

Reply:
This wasn't immediately clear to me when reading out of context, and I'm the one who wrote it. I think this is a good change to make personally.

Reply:
Alright, sounds good.